package.json

```diff
@@ -67,6 +67,7 @@
     "nx": "14.1.7",
     "postcss": "^8.4.14",
     "postcss-preset-env": "^7.7.1",
+    "postcss-replace": "1.1.3",
```

Comment:
ℹ️ Supports fixing paths to fonts when CSS is processed (see [`.postcssrc.yml`](https://github.com/TACC/tup-ui/pull/47/files#diff-b6e79afca41727ae4b6cb70403fa79f5abf4565638d06b40d9732fdd65a6f268)).